lib/dialog/index.jsx

```diff
@@ -21,25 +29,24 @@ export class Dialog extends Component {
           'dialog theme-color-bg theme-color-fg theme-color-border'
         )}
       >
-        {!hideTitleBar &&
-          onDone && (
-            <div className="dialog-title-bar theme-color-border">
-              <div className="dialog-title-side" />
-              <h2 className="dialog-title-text">{title}</h2>
-              <div className="dialog-title-side">
-                {!!onDone && (
-                  <button
-                    type="button"
-                    aria-label="Close dialog"
-                    className="button button-borderless"
-                    onClick={onDone}
-                  >
-                    Done
-                  </button>
-                )}
-              </div>
+        {!hideTitleBar && (
+          <div className="dialog-title-bar theme-color-border">
+            <div className="dialog-title-side" />
+            <h2 className="dialog-title-text">{title}</h2>
+            <div className="dialog-title-side">
+              {!!onDone && (
+                <button
+                  type="button"
+                  aria-label="Close dialog"
+                  className="button button-borderless"
+                  onClick={onDone}
+                >
```

Comment:
Hmm, I thought about adding a `Back` button in the upper left corner, but I dropped it for this iteration since it would complicate the state management.

My hunch is that most people don't do multiple imports in succession, but I guess we'll see in the usage data?

Reply:
Yeah sounds good 👍